packages/core/src/Asset/types/TokenMetadata.ts

```diff
@@ -30,7 +30,7 @@ export interface TokenMetadata {
    * When implementing the parser, it is recommended to also check 'image' and 'logo'
    * properties for backwards compatibility.
    */
-  icon?: string;
+  icon?: string | string[];
```

Comment:
Is this valid? Can't see anything about arrays in the [CIP](https://github.com/cardano-foundation/CIPs/pull/137/files#diff-f478f61a4b67bc7fee1d41bbb427028a7615beaacf726efc9289104e64f26ffaR64)

Reply:
Thanks, I've changed it